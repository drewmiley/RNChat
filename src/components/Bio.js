import React from 'react';

export default class Bio extends React.Component {
    constructor(props) {
        super(props);
        this.shouldComponentUpdate = false;
    }
    render() {
        return <div className='chatbot-bio'>
            <span>{this.props.bio}</span>
        </div>
    }
};