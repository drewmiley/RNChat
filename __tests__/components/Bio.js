import 'react-native';
import React from 'react';
import renderer from 'react-test-renderer';

import Bio from '../..//src/components/Bio';

describe('Bio', () => {

    describe('Rendering', () => {

        it('should match the snapshot', () => {
            const component = renderer.create(
                <Bio />
            );            
            expect(component).toMatchSnapshot();
        });

    });

});