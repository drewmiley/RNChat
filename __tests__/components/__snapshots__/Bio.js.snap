exports[`Bio Rendering should match the snapshot 1`] = `
<View>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail" />
</View>
`;
